www/js/metrics.js

```diff
@@ -443,6 +456,11 @@ angular.module('emission.main.metrics',['nvd3', 'emission.services', 'ionic-date
           var aggCount = results[2].aggregate_metrics;
```

Comment:
I have never really understood how the user values work here. if `results[0].user_metrics.length == 0`, then results[0].user_metrics is an empty array. So what does it mean to set the value `var userDuration = results[0].user_metrics;` here? Why do we need to treat first and not first differently?